src/utils.jl

```diff
@@ -302,6 +302,11 @@ Base.convert(::Type{Vector{T}}, rng::AbstractRange{S}) where {T<:Real,S<:Real} =
 
 Base.merge(a::AbstractVector, b::AbstractVector) = sort(unique(vcat(a,b)))
 
+# recursively merge kw-dicts, e.g. for merging extra_kwargs / extra_plot_kwargs in plotly)
+recursive_merge(x::AbstractDict...) = merge(recursive_merge, x...)
```

Comment:
Is this neccessary? Usually both dicts are only one level deep, or?

Reply:
I think we need it, because we want to merge the properties of respective axes. Just using `merge` on `plotattributes_out` would empty the rest of the arguments, which have been populated during `plotly_layout()`.

Maybe, I am wrong?

Reply:
How does this work out with multiple subplots? Something like
```julia
Plots.plot(rand(4,4), [[1,4,9,16]*10000, [0.5, 2, 4.5, 8], rand(4), rand(4)],
    layout = 4,
    include_mathjax = local_mathjax(),
    extra_plot_kwargs = KW(:yaxis => KW(:automargin => true), :xaxis => KW(:domain => "auto"))
)
```